ci/Jenkinsfile.qt-build

```
@@ -0,0 +1,224 @@
+                        docker build \
+                            --build-arg GOLANG_VERSION=1.23.10 \
+                            --build-arg NIM_VERSION=2.0.12 \
+                            --build-arg NIX_VERSION=2.24.11 \
+                            -t ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG}-ci .
+                    """
+                }
+            }
+        }
+
+        stage('Test Images') {
+            parallel {
+                stage('Test Qt Base Image') {
+                    steps {
+                        script {
+                            echo "Testing the Qt base image..."
+                            sh """
+                                docker run --rm \
+                                    ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG} \
+                                    "qmake --version"
+
+                                docker run --rm \
+                                    ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG} \
+                                    "ls -la /opt/qt/${params.QT_VERSION}/"
+                            """
+                        }
+                    }
+                }
+
+                stage('Test Complete Image') {
+                    steps {
+                        script {
+                            echo "Testing the image with nim/go tooling..."
+                            sh """
+                                docker run --rm \
+                                    ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG}-ci \
+                                    "qmake --version && nim --version && go version && echo 'All tools working!'"
+
+                                docker run --rm \
+                                    ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG}-ci \
+                                    "ls -la /home/jenkins/openssl-output/arm64-v8a/lib/"
+                            """
+                        }
+                    }
+                }
+            }
+        }
+
+
+        stage('Push to DockerHub') {
+            when {
+                expression { params.PUSH_TO_DOCKERHUB == true }
+            }
+            steps {
+                script {
+                    withCredentials([
+                        usernamePassword(
+                            credentialsId: 'dockerhub-statusteam-bot',
+                            usernameVariable: 'DOCKER_USER',
+                            passwordVariable: 'DOCKER_PASS'
+                        )
+                    ]) {
+                        sh """
+                            echo \$DOCKER_PASS | docker login -u \$DOCKER_USER --password-stdin
+                            docker push ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG}
+                            docker push ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG}-ci
+                            docker logout
+                        """
```

```suggestion
                        image.push()
                        image.push("${params.DOCKER_TAG}-ci")
```
What's the point of the `-ci` part?